### PR TITLE
Add favicon default value

### PR DESCRIPTION
**Pull request by nmacianx**
## Related Issues

Follow up of https://github.com/descope/auth-hosting/pull/555

## Description

💬 Adds a favicon default value to the env var

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -58,7 +58,9 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
-	const faviconUrl = process.env.REACT_APP_FAVICON_URL || '';
+	const faviconUrl =
```
Comment (dorsha) [inline src/App.tsx:61]: Just notice that existing customers might get console error if there is no favicon.
Can you check that?

Comment (nmacianx) [reply src/App.tsx:61]: no console errors - just checked @dorsha

Comment (dorsha) [reply src/App.tsx:61]: What about network error? 
Can you paste here a screenshot of the network when the URL does not exist?

Comment (nmacianx) [reply src/App.tsx:61]: image
image
image

Comment (dorsha) [reply src/App.tsx:61]: So this is a regression.

Comment (dorsha) [reply src/App.tsx:61]: Can we know if an image is there? Maybe we should have default image for all projects/apps.

Comment (nmacianx) [reply src/App.tsx:61]: @dorsha why would it be a regression?

Comment (nmacianx) [reply src/App.tsx:61]: we already have a default favicon in the app that is used 🤔 
i don't think there's a need to duplicate data and upload the same default favicon every time

Comment (dorsha) [reply src/App.tsx:61]: Today, customers don't have any network error.
And now after your change they will get a network error, right?

**Review by dorsha — APPROVED** [review]

**Pull request merged**